lib/src/revset.rs

```diff
@@ -2661,6 +2661,8 @@ pub fn format_remote_symbol(name: &str, remote: &str) -> String {
 
 #[cfg(test)]
 mod tests {
+    #![allow(clippy::cloned_ref_to_slice_refs)]
```

Comment:
is this the same thing as putting the attribute without the `!` on the line above?